### PR TITLE
Add docs for blurred background

### PR DESCRIPTION
**Pull request by rishavbhowmik**
- Modified docs of 'bg' param for image & video transformation.
- Update guide for cloudinary-rewriter for parameter `b` (background)

ToDo, once the feature is live:

- [x] Upload/copy <https://ik.imagekit.io/demo/img/pexels-nadezhda-moryak-6641295_u_VehrQDL.mp4?tr=orig> file to address `https://ik.imagekit.io/demo/sample2.mp4`.

### CODE REVIEW

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 1]
```diff
@@ -747,17 +747,17 @@ This is good for creating profile pictures.
 {% endtab %}
 {% endtabs %}
 
-### Background color - (bg)
+### Background - (bg)
 
-It is used to specify the background color that can be used along with some cropping strategies while resizing an image.
+It is used to specify the background that can be used along with some cropping strategies while resizing an image. Background can be any solid color or blurred image.
```
Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md]: "This feature allows you to define the background when resizing an image, and it works seamlessly with various cropping strategies. You can choose either a solid color or a blurred version of the same image as the background."

Bring the limitation message up here.

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 2]
```diff
@@ -772,6 +772,43 @@ URL - [https://ik.imagekit.io/demo/tr:h-700,w-700,cm-pad\_extract,bg-A94D34/medi
 {% endtab %}
 {% endtabs %}
 
+**Blurred background**
+
+Syntax: `blurred[_<BlurIntensity>][_<Brightness>]`
+
+| Optional parameter |                    Possible value |                                                                         Description |   Default |
+| -----------------: | --------------------------------: | ----------------------------------------------------------------------------------: | ---------: |
+|     Blur intensity | `auto` or integer from 0 to 100 |                                                              Set intensity of blur  |       'auto' |
```
Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md]: or an integer...

Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md]: Set the intensity of blur.

Run text through Grammarly or QuillBot once.

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 3]
```diff
@@ -772,6 +772,43 @@ URL - [https://ik.imagekit.io/demo/tr:h-700,w-700,cm-pad\_extract,bg-A94D34/medi
 {% endtab %}
 {% endtabs %}
 
+**Blurred background**
+
+Syntax: `blurred[_<BlurIntensity>][_<Brightness>]`
+
+| Optional parameter |                    Possible value |                                                                         Description |   Default |
```
Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md]: We shouldn't write optional parameter here, it should be explained in syntax description. Just write `Parameter` and `Values`.

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 4]
```diff
@@ -772,6 +772,43 @@ URL - [https://ik.imagekit.io/demo/tr:h-700,w-700,cm-pad\_extract,bg-A94D34/medi
...
+
+Limitation: Blurred background can only be used with `cm-pad_resize`
+
+Blur intensity `auto`: The intensity of blur is automatically adjusted based on the provided height (h) & width (w) in the transformation.
+
+{% tabs %}
+{% tab title="bg=blurred_12_N25" %}
+URL - [https://ik.imagekit.io/demo/tr:h-600,w-800,cm-pad\_resize,bg-blurred/medium\_cafe\_B1iTdD0C.jpg](https://ik.imagekit.io/demo/tr:h-600,w-800,cm-pad\_resize,bg-blurred/medium\_cafe\_B1iTdD0C.jpg)
+
+![cm-pad\_resize,bg-blurred](../../.gitbook/assets/medium_cafe-cm-pad\_resize,bg-blurred.png)
```
Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md:792]: What's with the `\` in transformation name and file paths?

File: features/image-transformations/resize-crop-and-other-transformations.md [hunk 5]
```diff
@@ -772,6 +772,43 @@ URL - [https://ik.imagekit.io/demo/tr:h-700,w-700,cm-pad\_extract,bg-A94D34/medi
...
+| Optional parameter |                    Possible value |                                                                         Description |   Default |
+| -----------------: | --------------------------------: | ----------------------------------------------------------------------------------: | ---------: |
+|     Blur intensity | `auto` or integer from 0 to 100 |                                                              Set intensity of blur  |       'auto' |
+|         Brightness | Integer from N255 to 255        |   Negative value to make background darker and positive value to make image lighter |            0 |
+
+Limitation: Blurred background can only be used with `cm-pad_resize`
+
+Blur intensity `auto`: The intensity of blur is automatically adjusted based on the provided height (h) & width (w) in the transformation.
+
+{% tabs %}
```
Comment (abhinavr4) [inline features/image-transformations/resize-crop-and-other-transformations.md:788]: We have to see how this tab looks with images linked here.

File: features/video-transformation/resize-crop-and-other-common-video-transformations.md [hunk 6]
```diff
@@ -290,16 +290,66 @@ URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=n-video\_thumbnail](https
 
 ### Background color - (bg)
```
Comment (abhinavr4) [inline features/video-transformation/resize-crop-and-other-common-video-transformations.md]: This should also change to `Background`

File: features/video-transformation/resize-crop-and-other-common-video-transformations.md [hunk 7]
```diff
@@ -290,16 +290,66 @@ URL - [https://ik.imagekit.io/demo/sample-video.mp4?tr=n-video\_thumbnail](https
...
 
-It is used to specify the background color in RGB Hex Code (e.g. FF0000). This is usually used with [pad\_resize](resize-crop-and-other-common-video-transformations.md#pad-resize-crop-strategy-cm-pad\_resize) cropping to control the color of extra background padding.
+It is used to specify the background for [crop-mode pad\_resize](resize-crop-and-other-common-video-transformations.md#pad-resize-crop-strategy-cm-pad\_resize) or border-radius (r). Background can be any solid color or blurred frame of the image or video.
```
Comment (abhinavr4) [inline features/video-transformation/resize-crop-and-other-common-video-transformations.md]: It is used to specify the background of the video when using pad_resize or border-radius. The background can be any solid colour or blurred frame of the video.

images is not needed here.

**Review by abhinavr4 — CHANGES_REQUESTED** [review]

File: .gitbook/assets/screenshot-cm-pad_resize-bg-A94D34.png
Comment (niveditn) [inline .gitbook/assets/screenshot-cm-pad_resize-bg-A94D34.png:1]: Screenshot 2023-09-26 at 6 56 45 PM

At first glance this seems like an image, not video. This should ideally be replaced with a screenshot where video player controls are visible

**Comment by niveditn** [comment]
> Modified docs of 'bg' param for image & video transformation.
> 
> ToDo, once the feature is live:
> 
>     * [ ]  Upload/copy https://ik.imagekit.io/demo/img/pexels-nadezhda-moryak-6641295_u_VehrQDL.mp4?tr=orig file to address `https://ik.imagekit.io/demo/sample2.mp4`.

@imagekitio Could we please upload this file to `/demo` ML?

File: .gitbook/assets/screenshot-cm-pad_resize-bg-A94D34.png
Comment (niveditn) [reply .gitbook/assets/screenshot-cm-pad_resize-bg-A94D34.png:1]: Also we should change this sample video to a more aesthetic one later. @rishavbhowmik

**Pull request merged**